### PR TITLE
fix: remove audit button

### PR DESCRIPTION
**Pull request by gloriateodoro**
## Checklist
- [x] /About - remove o botão de baixar auditoria

### CODE REVIEW

**Comment by bnd-operator** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **lpirola**
You can assign the PR to them by writing `/assign @lpirola` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=nossas%2Fnossas-org).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/nossas/nossas-org/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by bnd-operator** [comment]
:star: PR built and available in a preview **nossas-nossas-org-pr-24** [here](http://nossas-org-pr24.35.170.28.138.nip.io)

**Review by lpirola — APPROVED** [review]

**Comment by bnd-operator** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[lpirola](https://github.com/nossas/nossas-org/pull/24#pullrequestreview-674704882 "Approved")*

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=nossas%2Fnossas-org).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/nossas/nossas-org/blob/main/OWNERS)~~ [lpirola]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
